src/queryWizard/WizardGroup.js

```diff
@@ -0,0 +1,119 @@
+import F from 'futil-js'
+import _ from 'lodash/fp'
+import React from 'react'
+import { observer } from 'mobx-react'
+import { Dynamic, Flex } from '../layout'
+import InjectTreeNode from '../utils/injectTreeNode'
+import styles from '../styles'
+
+export let DefaultMissingTypeComponent = InjectTreeNode(({ node = {} }) => (
+  <div>
+    Type <b>{node.type}</b> is not supported (for key <i>{node.key}</i>)
+  </div>
+))
+
+let FieldDescription = observer(({ node }) => (
+  <div>
+    {node.fieldDescription} {node.typeDescription}
+  </div>
+))
+let WizardItem = ({
+  node,
+  tree,
```

Comment:
we should follow the convention you established on your QueryBuilder PR to call this root and not tree